keytools/generate_test.go

```diff
@@ -18,54 +18,41 @@ const (
 )
 
 func GenerateKeyFilesForTest(t *testing.T, curve string) (publicKey, privateKey []byte) {
-
 	publicFileName := "publicKeyFile"
 	privateFileName := "privateKeyFile"
-
 	GenerateSigningKeyPair(publicFileName, privateFileName, curve)
 
 	_, err := os.Stat(publicFileName)
-
 	assert.False(t, err != nil, "public key file not generated")
 
 	_, err = os.Stat(privateFileName)
-
 	assert.False(t, err != nil, "private key file not generated")
 
 	publicKey, err = utils.ReadKeyFromPemFile(publicFileName, utils.PublicKey)
-
 	if err != nil {
 		log.Fatal(err)
 	}
 
 	privateKey, err = utils.ReadKeyFromPemFile(privateFileName, utils.PrivateKey)
-
 	if err != nil {
 		log.Fatal(err)
 	}
 
 	os.Remove(publicFileName)
 	os.Remove(privateFileName)
-
-	return
-
+	return publicKey, privateKey
```

Comment:
What is the recommended style guide?

Reply:
Recommended style is to not just say `return` because a reader has to put an extra effort to see what instances we are returning.

Returning `return arg1, arg2` would be super helpful and avoids some confusion.